subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/inception/GradleInceptionPerformanceTest.groovy

```diff
@@ -82,7 +82,7 @@ class GradleInceptionPerformanceTest extends AbstractCrossVersionPerformanceTest
         runner.testProject = testProject
         runner.tasksToRun = ['help']
         runner.runs = runs
-        runner.args = extraGradleBuildArguments() + ["-Pgradlebuild.skipBuildSrcChecks=true"]
+        runner.args = extraGradleBuildArguments() + ["-buildSrcCheck=false"]
```

Comment:
Typo: `-PbuildSrcCheck=false`

The baseline Gradle may not support `buildSrcCheck`, right? So I guess we need to keep both arguments for now.

Reply:
Very good points! Thanks Bo